src/pages/Swap/AdvancedSwapMode/AdvancedSwapMode.styles.ts

```diff
@@ -0,0 +1,182 @@
+import styled, { css } from 'styled-components'
+
+export const BaseWrapper = styled.div`
+  background: #101019;
```

Comment:
don't we have a similar background color on theme? If so, it would be great to use it instead.

Reply:
unfortunately not, I tested with all theme bg-s and dark-s but there are no similar, bg6 and bg1 I think would be ok but they impact on swapbox background

Reply:
I will create a new theme bg9